it/uk/gov/hmrc/minorentityidentificationfrontend/controllers/CaptureUtrControllerISpec.scala

```diff
@@ -87,7 +87,7 @@ class CaptureUtrControllerISpec extends ComponentSpecHelper with AuthStub with S
 
         result must have(
           httpStatus(SEE_OTHER),
-          redirectUri(routes.CheckYourAnswersController.show(testJourneyId).url)
+          redirectUri(routes.CaptureOverseasTaxIdentifiersController.show(testJourneyId).url)
```

Comment:
I think the name of these tests will need updating from 'redirect to check your answers' -> 'redirect to overseas identifiers'